pom.xml

```diff
@@ -63,23 +63,23 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.7.1</jenkins.version>
```

Comment:
I also would prefer 2.7.3. For Pipeline plugins we generally depend on the most recent update in a given LTS line. This ensures that tests etc. are incorporating known fixes.